CI/appveyor.set-build-info.ps1

```diff
@@ -13,7 +13,8 @@ if ($Env:APPVEYOR_REPO_TAG -eq "false") {
   } else {
     if ($Env:MUDLET_VERSION_BUILD -eq "-ptb") {
       $Script:Date = Get-Date -Format "yyyy-MM-dd"
-      $Env:MUDLET_VERSION_BUILD = "$Env:MUDLET_VERSION_BUILD-$Date"
+      $Script:Short_Commit = $Script:Commit.Substring(0, 5)
+      $Env:MUDLET_VERSION_BUILD = "$Env:MUDLET_VERSION_BUILD-$Date-$Short_Commit"
     } else {
       $Script:Commit = git rev-parse --short HEAD
```

Comment:
Doesn't this line have to be pulled out of the current else case, so it can be used for Substring() in the if case?